tests/java-tests/src/test/java/test/AdapterTest.java

```diff
@@ -3,21 +3,22 @@
 import com.apollographql.apollo3.api.Adapters;
 import com.apollographql.apollo3.api.CustomScalarAdapters;
 import com.apollographql.apollo3.api.NullableAdapter;
-import com.apollographql.apollo3.api.StringAdapter;
 import com.apollographql.apollo3.api.json.BufferedSourceJsonReader;
+import kotlin.jvm.Throws;
 import okio.Okio;
 import org.junit.Test;
 
 import java.io.ByteArrayInputStream;
+import java.io.IOException;
 
 import static com.google.common.truth.Truth.assertThat;
 
 public class AdapterTest {
   @Test
-  public void nonNullString() {
+  public void nonNullString() throws IOException {
     String json = "\"test\"";
     BufferedSourceJsonReader jsonReader = new BufferedSourceJsonReader(Okio.buffer(Okio.source(new ByteArrayInputStream(json.getBytes()))));
-    String result = StringAdapter.INSTANCE.fromJson(jsonReader, CustomScalarAdapters.Empty);
+    String result = Adapters.StringAdapter.fromJson(jsonReader, CustomScalarAdapters.Empty);
```

Comment:
Indeed this looks nicer from Java! Should we recommend using top level`val` instead of objects in Contributing.md?

Reply:
Good idea 👍 . I'll add a line

Reply:
Added there: https://github.com/apollographql/apollo-android/pull/3607/commits/f2d3f5e0d31e871401d6a6d02d23fe305ceb170c

Also took this opportunity to mention doc/tutorial/tweet in the release process